### DISCUSSION TITLE
Refresh the publish version

### DISCUSSION THREAD

**Discussion by dew-automatisation** [Budibase/budibase#13203] [Help] [answered]
Hello

I have a problem. I create a page with "logs - Grid block" and publish it.
The problem is when I add data to this Budibase database, it is not updated on the published page. I have to republish the page each time. 
How can I avoid having to republish the page?

Thanks

**Comment by ConorWebb96** [comment]
If you are adding data via the builder instead of on your published app you will need to publish after making changes for it to sync across from the builder.

You can use the grid component within your app to have the same edibility you get from using the builder.

**Comment by dew-automatisation** [comment]
Ok thanks, just I add data by the API

**Reply by ConorWebb96** [reply, marked as answer]
Have you tried taking the _dev out of your app url?

**Reply by dew-automatisation** [reply]
Thanks it was that :)
